frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -42,6 +42,7 @@
         button.is-outlined(
           key="back"
           v-if='!isConfirmation'
+          type="button"
```

Comment:
@taoeffect and @mmbotelho the bug that closes the modal when pressing enter on mincome input is fixed, as well the error about "next" method not existing.

FYI, the cause is a common bug when dealing with forms. By default, a button has type 'submit' instead of 'button'.  So, when inside a `<form>` and you press enter, it will find the first 'submit' button and trigger it. In our case, our first submit button was this one (Cancel / Back), that's why the modal closes.

Now the first 'submit' button is "Next" or "Create Proposal" and all works fine ;)

cc @pieer